### PR TITLE
Setup Prettier

### PR DESCRIPTION
**Pull request by mikhail-shkaralevich**
# Ticket Link

[GROUPINGS-1684](https://uhawaii.atlassian.net/browse/GROUPINGS-1684)

# List of squashed commits

-   Set up Prettier

# Confluence Page

[Prettier set up](https://uhawaii.atlassian.net/wiki/x/KIBMLQ)

# Test Checklist

-   [x] Unit Tests Passed
-   [x] Integration Tests Passed
-   [x] General Visual Inspection

### CODE REVIEW

**Comment by mikhail-shkaralevich** [comment]
I ran jest on the branch and all of tests have passed.
image

File: ui/.prettierrc
Comment (JorWo) [inline ui/.prettierrc:1]: I think this file needs to be formatted by Prettier haha.

**Review by JorWo — APPROVED** [review]
Everything else looks good! Will merge after fixing my small nitpick.

**Review by mhodgesatuh — APPROVED** [review]

**Comment by JorWo** [comment]
There seems to be a merge conflict now too.

**Comment by mikhail-shkaralevich** [comment]
> There seems to be a merge conflict now too.

It is resolved now. Thanks

**Pull request merged**